src/components/requestDetail/NewComment.vue

```diff
@@ -1,46 +1,27 @@
 <script lang="ts" setup>
-import { ArrowPathIcon } from '@heroicons/vue/24/solid'
-
-import type { Comment } from '/@/lib/apis'
+import { useRequestDetailStore } from '/@/stores/requestDetail'
 
 import MarkdownTextarea from '/@/components/shared/MarkdownTextarea.vue'
 import SimpleButton from '/@/components/shared/SimpleButton.vue'
 
 import { useComment } from './composables/useComment'
 
-interface Props {
-  requestId: string
-}
-
-const props = defineProps<Props>()
-const emit = defineEmits<{
-  (e: 'pushComment', value: Comment): void
-}>()
+const requestDetailStore = useRequestDetailStore()
```

Comment:
issueにもしてたと思うけど、こういうのはこまめにやっていっていいと思います
```suggestion
const { request } = storeToRefs(useRequestDetailStore())
if (!request.value) throw new Error('hoge')
```